src/qt/pivx/coldstakingwidget.cpp

```diff
@@ -660,6 +657,7 @@ void ColdStakingWidget::onDeleteClicked() {
         label = index.data(Qt::DisplayRole).toString();
     }
     inform(label + tr(" blacklisted from staking"));
+    updateStakingTotalLabel();
```

Comment:
Minor thing, move the `updateStakingTotalLabel()` before the `inform` or it will not be updated up until the snackbar disappear.